librz/core/cmd/cmd_print.c

```diff
@@ -6070,14 +6128,13 @@ RZ_IPI int rz_cmd_print(void *data, const char *input) {
 			if (l > 0) {
 				int h, w = rz_cons_get_size(&h);
 				int colwidth = rz_config_get_i(core->config, "hex.cols") * 2;
-				core->print->width = (colwidth == 32) ? w : colwidth; // w;
```

Comment:
I see this is used only in very few places... could you double check if `print->width` can be removed completely from RzPrint please?

Reply:
i will remove it in another PR.